==== TEST RUN LOG ====
Test Name      : diamondgotcat.ypsh
Test File      : /Users/runner/work/AutoTestingSystem/AutoTestingSystem/tests/diamondgotcat.ypsh.py
Command        : /Library/Frameworks/Python.framework/Versions/3.12/bin/python /Users/runner/work/AutoTestingSystem/AutoTestingSystem/tests/diamondgotcat.ypsh.py
Start (UTC)    : 2025-08-27T05:04:28.714813+00:00
Epoch (ns)     : 1756271068714799000
---- ENVIRONMENT ----
python_executable: /Library/Frameworks/Python.framework/Versions/3.12/bin/python
python_version   : 3.12.10 (v3.12.10:0cc81280367, Apr  8 2025, 08:46:59) [Clang 13.0.0 (clang-1300.0.29.30)]
platform         : macOS-14.7.6-arm64-arm-64bit
cwd              : /Users/runner/work/AutoTestingSystem/AutoTestingSystem
tests_dir        : /Users/runner/work/AutoTestingSystem/AutoTestingSystem/tests
os               : macos
arch             : arm64
----------------------
---- STDOUT ----

---- STDERR ----
Traceback (most recent call last):
  File "/Users/runner/work/AutoTestingSystem/AutoTestingSystem/tests/diamondgotcat.ypsh.py", line 89, in <module>
    main()
  File "/Users/runner/work/AutoTestingSystem/AutoTestingSystem/tests/diamondgotcat.ypsh.py", line 80, in main
    execFilePath = download_and_extract(stableTag, osname, arch)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/work/AutoTestingSystem/AutoTestingSystem/tests/diamondgotcat.ypsh.py", line 41, in download_and_extract
    response.raise_for_status()
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/requests/models.py", line 1026, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://github.com/YPSH-DGC/YPSH/releases/download/v6.5.1%0A/YPSH-macos-arm64.zip
-----------------
End   (UTC)    : 2025-08-27T05:04:29.290447+00:00
Duration (s)   : 0.575634
Return Code    : 1
======================
